test/new/db/tools/r2r

```diff
@@ -44,23 +43,22 @@ EXPECT_ERR=<<EOF
 EOF
 RUN
 
-# TODO: Enable when r2r.js and r2r.v are dead
-#NAME=bin with space in filename (multiple files)
-#FILE=<<EOF
-#-
-#-
-#../bins/elf/_Exit (42)
-#-
-#-
-#EOF
-#CMDS=ob
-#EXPECT=<<EOF
-#0 3 x86-64 ba:0x00000000 sz:512 malloc://512
-#1 4 x86-64 ba:0x00000000 sz:512 malloc://512
-#2 5 x86-32 ba:0x08048000 sz:13073 ../bins/elf/_Exit (42)
-#3 6 x86-64 ba:0x00000000 sz:512 malloc://512
-#4 7 x86-64 ba:0x00000000 sz:512 malloc://512
-#EOF
-#EXPECT_ERR=<<EOF
-#EOF
-#RUN
+NAME=bin with space in filename (multiple files)
+FILE=<<EOF
+-
```

Comment:
or this

Reply:
fixed this by removing it since it's unnecessary 77a1eb79ee041df4ed646efc7657bcfa6125b466